src/Modal.js

```diff
@@ -55,7 +55,7 @@ const Modal = React.createClass({
         className={classNames(this.props.className, classes)}
         onClick={this.props.backdrop === true ? this.handleBackdropClick : null}
         ref="modal">
-        <div className={classNames(dialogClasses)}>
+        <div className={classNames(this.props.dialogClassName, dialogClasses)}>
```

Comment:
Should we not have validation on `dialogClassName`, at least to ensure that this is a string?